src/db.ts

```diff
@@ -25,3 +26,70 @@ async function run() {
 export const dbConnect = async () => {
     await run().catch(console.dir);
 }
+
+class dbHandler {
+    private uri: string = process.env.MONGO_URI ?? 'mongodb://localhost:27017/myapp';
```

Comment:
why the ??, MONGO URI should already be there. duplicating the variable uri from the top